Engine/src/Engine/Application.cpp

```diff
@@ -1,10 +1,46 @@
 #include "Application.h"
 #include "Scene.h"
+#include "ResourceManager.h"
 
 namespace Engine
 {
 	void Application::Run()
 	{	
+		nlohmann::json data = {
```

Comment:
This seems more like a test json. If you verified that it works, I imagine this should be removed.

Reply:
whoops